### PR TITLE
Add support for RDS

### PR DESCRIPTION
**Pull request by LMJW**
*Issue #, if available:* 

Did not find a particular issue for supporting RDS, but I guess this is probably planned. If RDS support is not needed at the moment, I will close this pull request.

*Description of changes:* 
- clone the smithy model json for rds from GO SDK https://github.com/aws/aws-sdk-go-v2/blob/main/codegen/sdk-codegen/aws-models/rds.2014-10-31.json
- create a "helloword example" for RDS

**Note**, there is another [rds-data](https://github.com/aws/aws-sdk-go-v2/blob/main/codegen/sdk-codegen/aws-models/rdsdata.2018-08-01.json) smithy model. I will work on to add it if this PR is okay to merge.

*Test:*
- `./gradlew :aws:sdk:assemble` SUCCESS
- `./gradlew :aws:sdk:test` SUCCESS
- `./gradlew :aws:sdk:cargoCheck` SUCCESS

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: aws/sdk/examples/rds-helloworld/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+rds = {package = "aws-sdk-rds", path = "../../build/aws-sdk/rds"}
+
+tokio = {version = "1", features = ["full"]}
+
+# used only for static endpoint configuration:
+http = "0.2.3"
```
Comment (rcoh) [inline aws/sdk/examples/rds-helloworld/Cargo.toml:15]:
```suggestion
```
(unused here)

File: aws/sdk/examples/rds-helloworld/Cargo.toml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+[package]
```
Comment (rcoh) [inline aws/sdk/examples/rds-helloworld/Cargo.toml:1]: can you rename that file to just `rds` to match the other files?

**Review by rcoh — COMMENTED** [review]
couple of small things, but looks good. Thanks!

**Comment by rcoh** [comment]
Fixes aws-sdk-rust#36

File: aws/sdk/examples/rds-helloworld/Cargo.toml (hunk 1 above)
Comment (LMJW) [reply aws/sdk/examples/rds-helloworld/Cargo.toml:15]: Yes. I will remove this dependency

File: aws/sdk/examples/rds-helloworld/Cargo.toml (hunk 2 above)
Comment (LMJW) [reply aws/sdk/examples/rds-helloworld/Cargo.toml:1]: Yeah, I will update `rds.2014-10-31.json` tobe `rds.json`

**Comment by LMJW** [comment]
Hi @rcoh Please let me know if you have other comments. :)

**Review by rcoh — APPROVED** [review]
LGTM! Can you please just confirm that you ran the example and it worked?

**Comment by LMJW** [comment]
Yeah, it worked. Let me try recreate a instance and get the output quickly.

**Comment by LMJW** [comment]
The example print out is 

```
     Running `/Users/xxx/rust/smithy-rs/aws/sdk/build/aws-sdk/target/debug/rds-helloworld`
DB instance identifier: "test-m5"
DB instance class: "db.m5.large"
DB instance engine: "postgres"
DB instance status: "available"
DB instance endpoint: Endpoint { address: Some("test-m5.c6h6hf3yffse.us-east-1.rds.amazonaws.com"), port: 5432, hosted_zone_id: Some("Z2Rxxxxx") }
```

**Pull request merged**